.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,16 @@
 ## Description
-
-<!--- Describe your changes in detail. -->
+<!--- Describe your changes in detail. The sections suggested are intended to make -->
+<!--- it easy to create a descriptive PR that is easy to review. Change as needed! --> 
 
 ## Motivation / Context
 <!--- Why is this change required? What problem does it solve? -->
-<!--- If it fixes, or is related to, an open issue, link to the issue here. -->
+<!--- If it fixes, or is related to an open issue, link to the issue here. -->
 
 ## Testing Instructions / How This Has Been Tested
-<!--- Describe in detail how you tested your changes. -->
-<!--- Include details of your testing environment, and the tests you ran. -->
+<!--- Describe how you tested your changes and/or how a reviewer can test your changes. -->
 
 ## Screenshots
-<!--- If applicable. --->
+<!--- Would including screenshots be helpful to the reviewer? --->
 
 ## Documentation
-<!--- Do any of the changes affect the accuracy of existing documentation? --->
-<!--- Do they warrant new sections within the current documentation? --->
-
-## PR Author Checklist
-
-- [ ] My PR description includes a reference to a ticket or other "motivation" for these changes.
-- [ ] My PR description includes details of how I tested these changes.
-- [ ] My PR description includes testing instructions for reviewers with links to the testing environment, if applicable.
-- [ ] My code includes documentation updates to existing documentation or new sections if applicable.
+<!--- Did any of the changes cause documentation to need to be updated? --->
```

Comment:
Can we improve this question a bit?
```suggestion
<!--- Do any of the changes warrant documentation updates? --->
```

Take it to leave it, though. This is not a deal-breaker.